### PR TITLE
Fix aws deployment

### PR DESCRIPTION
**Pull request by clemsos**
# Description

We add a prod stage to the docker file so we can produce a Docker image with the built version of the app.  

We use Heroku to deploy the built image from CI

Also we take this opportunity to make some cleaup in CI config file syntax

# Issues

Relates ot #7515

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by clemsos** [comment]
The CI has been updated to deploy locksmith to heroku.

File: .circleci/config.yml [hunk 1]
```diff
@@ -386,11 +206,35 @@ jobs:
...
+    parameters:
+      service:
+        description: "The service/app to build"
+        type: string
+      app-name:
+        description: "The Heroku app name to which the application should be deployed"
+        type: string
+    steps:
+      - checkout
+    # - run:
```
Comment (julien51) [inline .circleci/config.yml:225]: Leftover?

File: .circleci/config.yml [hunk 2]
```diff
@@ -386,11 +206,35 @@ jobs:
...
+    #       command: scripts/monorepo.sh locksmith $CIRCLE_BRANCH
+      - run:
+          name: Show docker info
```
Comment (julien51) [inline .circleci/config.yml:228]: same?

**Review by julien51 — APPROVED** [review]

**Comment by julien51** [comment]
ok, merging to test... we can still clean up after.

**Pull request merged**